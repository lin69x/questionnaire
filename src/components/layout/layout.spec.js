import React from 'react';
import Enzyme from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
import Layout from './layout';
import renderer from 'react-test-renderer';

Enzyme.configure({ adapter: new EnzymeAdapter() });

const props = {
  children: "test child",
};

describe('<Card/>', () => {
  it('should match snapshot', () => {
    const renderedValue = renderer.create(<Layout { ...props }/>).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });
});
