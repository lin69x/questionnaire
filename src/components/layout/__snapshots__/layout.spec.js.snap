// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card/> should match snapshot 1`] = `
<div
  className="container-fluid"
>
  test child
</div>
`;
